catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -236,32 +290,92 @@ public DeleteResponse version(DeleteResponse deleteResponse)
         }
         setSkipFlag(deleteResponse);
 
-        List<Metacard> deletedMetacards = deleteResponse.getDeletedMetacards()
+        if (LOGGER.isTraceEnabled()) {
+            LOGGER.trace("Versioning Deleted Metacards " + deleteResponse);
+        }
+
+        List<Metacard> originalMetacards = deleteResponse.getDeletedMetacards()
                 .stream()
                 .filter(isNotVersionNorDeleted)
                 .collect(Collectors.toList());
 
+        if (originalMetacards.isEmpty()) {
+            LOGGER.trace("No deleted metacards applicable to versioning");
+            SecurityLogger.audit("Skipping versioning deleted metacards with ids: "
```

Comment:
Should this be parameterized? I see in previous logging that isTraceEnabled is used if the processing to produce the logging message can be heavy. Should we consider using it here?